deviations/models.py

```diff
@@ -86,6 +86,19 @@ class SubmissionRuleDeviation(UrlMixin, models.Model):
         verbose_name=_('LABEL_SUBMITTER'),
         on_delete=models.CASCADE,
     )
+    granter = models.ForeignKey(UserProfile,
+        verbose_name=_('LABEL_GRANTER'),
+        on_delete=models.CASCADE,
+        related_name='+',
```

Comment:
`submitter` already used the automatically generated related names (`deadlineruledeviation_set` and `maxsubmissionsruledeviation_set`). I could've used `related_name="granted_%(class)_set"` here to avoid the name conflict, but I'm not sure if these related names would even be used ever, since these fields mostly exist for edit tracking purposes.

Reply:
Disabling the reverse relation is good!